### PR TITLE
feature/statsdesign: Simplifying design to a single row per user stat

### PR DESCRIPTION
**Pull request by nramirez**
Moving Stats redesign to its own PR https://github.com/kajahno/elprimobot/pull/50

Manually tested these changes for daily and weekly stats

* Not including bots stats in the update
* Single-line stats per username
image
image

### CODE REVIEW

File: api/stats.js [hunk 1]
```diff
@@ -1,24 +1,25 @@
...
+    https://discord.com/developers/docs/reference#snowflakes
+*/
+const getSnowflakeFromDay = (days) => {
+    const d = new Date();
+    d.setHours(0, 0, 0, 0);
+    d.setDate(d.getDate() + days);
+    return SnowflakeUtil.generate(d);
+};
+
+const BOTS = ["elprimobot", "ChillBot"];
```
Comment (kajahno) [inline api/stats.js:15]: nit: this could be an environment variable

**Review by kajahno — COMMENTED** [review]
LGTM

File: api/stats.js (hunk 1 above)
Comment (nramirez) [reply api/stats.js:15]: would this change for env?
Do you think this should just be another config property but not necessarily change per environment?

I'll move it to the config, not sure if there's a clear benefit so far.

Comment (nramirez) [reply api/stats.js]: done, FYI: added a new property BOTS and it is expecting a comma-separated list of names, it defaults to "elprimobot" however, for the local environment you want to include ["elprimobot","ChillBot"]

**Review by kajahno — APPROVED** [review]

**Pull request merged**